src/lib/VTXSPI/devVTXSPI.cpp

```diff
@@ -351,8 +364,9 @@ static int timeout()
         return DURATION_NEVER;
     }
 
-    if (!hwTimer::isTick) // Only run spi and analog reads during rx free time.
+    if ((hwTimer::running && !hwTimer::isTick) || stopVtxMonitoring)
```

Comment:
`!hwTimer::isTick` might not be required now that the VTx has its own SPI.  But it doesn't hurt to leave it.